test/unit/component/helpers/validators.test.ts

```diff
@@ -63,3 +63,15 @@ describe('validate opacity', () => {
     expect(ensureOpacityValue(input)).toEqual(expected);
   });
 });
+
+describe('validate stroke width', () => {
+  it.each`
+    input                      | expected
+    ${undefined}               | ${1}
+    ${12}                      | ${12}
+    ${-50}                     | ${1}
+    ${Number.MAX_SAFE_INTEGER} | ${Number.MAX_SAFE_INTEGER}
```

Comment:
**suggestion:** can we try to use a larger number? otherwise, I don't get why we put an upper bound in the validator

Reply:
we decided to decreased the max allowed value.